docs/remote-signing.md

```diff
@@ -136,7 +151,132 @@ Input an optional address look-ahead used to scan for used keys (default 2500):
 ```
 
 Alternatively a script can be used for initializing the watch-only wallet
-through the RPC interface as is described in the next section.
+through the RPC interface as is described in the
+[section below](#Example-initialization-script).
+
+### Outbound remote signer example
+
+To set up an outbound remote signer, the set up process needs to be made in 4
+steps:
+
+1. Start the signer node and export the `xpub`s of the wallet.
+2. Start watch-only node and initialize the watch-only wallet, and generate
+a macaroon + TLS certificate for the node.
+3. Update the configuration for the signer node, to include the watch-only
```

Comment:
Hmm, wondering how we can compress these last two steps to streamline the set up a bit. 

Why's the 4th step required? At step two, can we already set the signer type to outbound, relying on the back off logic to wait long enough for all the steps to be completed? 

Zooming out to 10k feet, I think an ideal UX would be something like:
  * User scans QR code generated by watch only node. 
  * Set up is complete, and app can be used to drive the node. 

Or you replace the QR code with a single RPC message sent to some service. 

Either way, requiring a restart really mucks up the flow IMO (making it harder to automate provisioning). It'd be best if we can figure out a way to get around that.

Reply:
With the latest changes to the CLI in, we should be able to skip some of the checks if we derive the macaroon directly from a root key. Would be nice to update the docs for that.